src/SAWScript/Yosys/Cell.hs

```diff
@@ -284,7 +284,21 @@ primCellToMap sc c args = case c ^. cellType of
     resPair <- liftIO $ SC.scApplyAll sc scFoldr [bool, accTy, swidth, fun, defaultAcc, ts]
     res <- liftIO $ SC.scPairRight sc resPair
     output $ CellTerm res (connWidthNat "A") (connSigned "Y")
-  -- "$bmux" -> _
+  CellTypeBmux -> do
```

Comment:
If there is an online reference that you could link to that helps explain how you figured out this implementation, that would be quite helpful. (If there isn't one, don't worry about it.)

Reply:
commit message updated